python/ray/autoscaler/kuberay/ray-cluster.complete.yaml

```diff
@@ -155,6 +157,12 @@ spec:
     rayStartParams:
       node-ip-address: $MY_POD_IP
       block: 'true'
+      # Use `resources` to optionally specify custom resource annotations for the Ray node.
+      # The value of `resources` is a string-integer mapping.
+      # Currently, `resources` must be provided in the unfortunate format demonstrated below.
+      # Moreover, "CPU" and "GPU" should NOT be included in the `resources` arg.
+      # (Use `num-cpus` and `num-gpus` rayStartParams instead.)
+      resources: '"{\"Custom2\": 5, \"Custom3\": 1}"'
```

Comment:
Gave the worker type some custom resources so we can test that autoscaling works with custom resources.